docs/configuration_file.md

```diff
@@ -137,6 +137,7 @@ Some options are also mandatory.
 
 * `can_orchestrate` (*optional*, `boolean`) — allows using orchestrator build, default is false
 
+* `info_url_format` (*optional*, `string`) — format for `url` dockerfile label, you can use `{label_name}` with any label, and if the label is defined it will be substituted.
```

Comment:
Now this is using .format() let's make the documentation clear on that point: "format for `url` Dockerfile label, used as a Python format string; replacement field keywords are label names, and they will be replaced with the value of the named label"